src/lib/select/select.spec.ts

```diff
@@ -1362,7 +1369,8 @@ describe('MdSelect', () => {
   selector: 'basic-select',
   template: `
     <div [style.height.px]="heightAbove"></div>
-    <md-select placeholder="Food" [formControl]="control" [required]="isRequired">
+    <md-select placeholder="Food" [formControl]="control" [required]="isRequired"
+      [tabIndex]="tabIndexOverride">
```

Comment:
I have been asking in https://github.com/angular/material2/issues/3368 about it - I thought about supporting both (for convenience).